tox.ini

```diff
@@ -3,20 +3,20 @@ envlist = py27, py27-tornado23, py27-tornado31, docs, flake8
 
 [testenv]
 sitepackages = true
-commands = nosetests -v --with-xunit --xunit-file=xunit-{envname}.xml --with-coverage --cover-package=mopidy
+commands = py.test --junit-xml=xunit-{envname}.xml --cov=mopidy
```

Comment:
From my experience with `nose --processes=12` I would suspect the
introspection in the deprecated property call. Try short circuiting that
check and see if it helps.

On Thu, Mar 5, 2015 at 9:09 AM, Stein Magnus Jodal <notifications@github.com
> wrote:

> In tox.ini
> <https://github.com/mopidy/mopidy/pull/1024#discussion_r25845323>:
>
> > @@ -3,20 +3,20 @@ envlist = py27, py27-tornado23, py27-tornado31, docs, flake8
> >
> >  [testenv]
> >  sitepackages = true
> > -commands = nosetests -v --with-xunit --xunit-file=xunit-{envname}.xml --with-coverage --cover-package=mopidy
> > +commands = py.test --junit-xml=xunit-{envname}.xml --cov=mopidy
>
> There's seems to be some issue with our test suite making py.test -n 6
> for example as slow as or slower than -n 1. I'll add pytest-xdist to the
> deps list in tox.ini too, and add {posargs} to the command. That way, we
> can experiment with e.g. tox -e py27 -- -n 6. When it works properly, we
> can make parallel execution the default.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mopidy/mopidy/pull/1024/files#r25845323>.
>